src/DynamoCoreWpf/ViewModels/Watch3D/DynamoLineGeometryModel3D.cs

```diff
@@ -13,6 +13,17 @@ protected override SceneNode OnCreateSceneNode()
         {
             return new DynamoLineNode();
```

Comment:
Don't we have to set `{ Material = material; }` in the constructor of `DynamoLineNode` as well?

Reply:
yes, will fix that, thanks for catching it.